### PR TITLE
feat: remove file flag from iac tests

### PR DESCRIPTION
**Pull request by almog27**
- [X] Ready for review
- [X] Follows [CONTRIBUTING](https://github.com/snyk/snyk/blob/master/.github/CONTRIBUTING.md) rules
- [x] Reviewed by Snyk internal team

#### What does this PR do?
Remove `--file` flag for IaC tests so the new command will be
`snyk iac test {file_name}`

### CODE REVIEW

File: src/lib/detect.ts [hunk 1]
```diff
@@ -144,22 +144,18 @@ export function detectPackageManager(root: string, options) {
...
-    throw UnsupportedLocalFolderIacError();
+  if (isLocalFolder(root)) {
+    debug('Iac - folder case ' + root);
+    throw OnlySupportLocalFileIacError();
   }
 
-  if (!fs.existsSync(root)) {
-    throw UnsupportedLocalFolderIacError();
+  if (localFileSuppliedButNotFound(root, '.') || !fs.existsSync(root)) {
+    throw OnlySupportLocalFileIacError();
```
Comment (orkamara) [inline src/lib/detect.ts:158]:
```suggestion
    throw SupportLocalFileOnlyIacError();
```

**Comment by anthogez** [comment]
> Fails
> 🚫	
> Commit ["Code review comments fixes"](https://api.github.com/repos/snyk/snyk/git/commits/bd50de90572e98138bfa96b82ef7d8601220c024) is not a valid commitizen message. See [Contributing page](https://github.com/snyk/snyk/blob/master/.github/CONTRIBUTING.md#commit-types) with required commit message format.
> 
> Generated by 🚫 [dangerJS](https://danger.systems/js) against [bd50de9](https://github.com/snyk/snyk/commit/bd50de90572e98138bfa96b82ef7d8601220c024)

Hey, please update the commit from `Code review comments fixes` to our code review conventions e.g. `fix: applying code review requests` thanks! :)

**Comment by anthogez** [comment]
> Fails
> 🚫	
> Commit ["Code review comments fixes"](https://api.github.com/repos/snyk/snyk/git/commits/bd50de90572e98138bfa96b82ef7d8601220c024) is not a valid commitizen message. See [Contributing page](https://github.com/snyk/snyk/blob/master/.github/CONTRIBUTING.md#commit-types) with required commit message format.
> 
> Generated by 🚫 [dangerJS](https://danger.systems/js) against [bd50de9](https://github.com/snyk/snyk/commit/bd50de90572e98138bfa96b82ef7d8601220c024)

Hey, please update the commit from `Code review comments fixes` to our code commit conventions e.g. `fix: applying code review requests`.
But.. on this specific, we are dealing also with public release notes, so is better if you squash the commits into a single one
and just leave a single commit message e.g. `feat: removing file flag from iac test cmd` to give more context to our users, thanks :)

File: src/lib/errors/unsupported-options-iac-error.ts [hunk 2]
```diff
@@ -1,7 +1,8 @@
 import { CustomError } from './custom-error';
 
-export function UnsupportedLocalFolderIacError() {
-  const errorMsg = 'iac test option currently support a single local file only';
+export function OnlySupportLocalFileIacError() {
+  const errorMsg =
+    'iac test option currently supports a single local file only';
```
Comment (anthogez) [inline src/lib/errors/unsupported-options-iac-error.ts:5]: I suggest `Iac test option currently supports only a single local file`

File: test/acceptance/cli-args.test.ts [hunk 3]
```diff
@@ -73,10 +73,28 @@ test('snyk test command should fail when iac file is not specified', (t) => {
   });
 });
```
Comment (anthogez) [inline test/acceptance/cli-args.test.ts:75]: Would be nice to have a test that confirms, which the error containing this message `Iac test option currently supports only a single local file` is actuality being sent/throwed

File: test/acceptance/cli-args.test.ts [hunk 4]
```diff
@@ -73,10 +73,28 @@ test('snyk test command should fail when iac file is not specified', (t) => {
...
 
+test('snyk test command should fail when iac --file is specified', (t) => {
```
Comment (anthogez) [inline test/acceptance/cli-args.test.ts:76]: Would be nice to have a test that confirms, which the error containing this message `Iac test option currently supports only a single local file` is actuality being sent/throwed.

Comment (almog27) [reply test/acceptance/cli-args.test.ts:76]: There is - in the main commit

Comment (anthogez) [reply test/acceptance/cli-args.test.ts:76]: Please review

> 
>   1) test/acceptance/cli-test/cli-test.acceptance.test.ts Languages Iac (Kubernetes) `iac test - no file` shows err:
>      Error: shows err
>       at Test.<anonymous> (test/acceptance/cli-test/cli-test.iac-k8s.spec.ts:38:11)
>       at runMicrotasks (<anonymous>)
> 
>   2) test/acceptance/cli-test/cli-test.acceptance.test.ts Languages Iac (Kubernetes) `iac test - not a real dir` shows err:
>      Error: shows err
>       at Test.<anonymous> (test/acceptance/cli-test/cli-test.iac-k8s.spec.ts:56:11)
>       at runMicrotasks (<anonymous>)
> 
>   3) test/acceptance/cli-args.test.ts snyk test command should fail when iac file is not specified correct error output:
>      Error: correct error output
>       at test/acceptance/cli-args.test.ts:68:7
> 
>   4) test/acceptance/cli-args.test.ts snyk test command should fail when iac --file is specified correct error output:
>      Error: correct error output
>       at test/acceptance/cli-args.test.ts:84:9

Comment (anthogez) [reply test/acceptance/cli-args.test.ts:76]: @almog27 yes sorry, just checked again  🤦 ahahaha :))
https://github.com/snyk/snyk/pull/1312/commits/ee83abf74562021fbf4885477c8b05de0ab234ca#diff-3efc9131f0a898c3cef25152ef04be5eL70

Comment (almog27) [reply test/acceptance/cli-args.test.ts:76]: The tests now run successfully

**Pull request merged**